src/routes.py

```diff
@@ -88,15 +88,20 @@ def topics() -> str | Response:
     return redirect(url_for("routes.login"))
 
 
-@bp.route("/topics/create")
-def create_topic() -> str:
+@bp.route("/topics/create", methods=["POST", "GET"])
+def create_topic() -> str | Response:
     """Handle create topic page."""
+    form = TopicForm()
```

Comment:
Let's move this variable definition closer to the place where it's actually used.